Tools/autotest/pysim/util.py

```diff
@@ -291,7 +305,9 @@ def start_SITL(binary,
             f.write("disable\n")
         f.write("r\n")
         f.close()
-        if os.environ.get('DISPLAY'):
+        if sys.platform == "darwin" and os.getenv('DISPLAY'):
+            cmd.extend(['gdb', '-x', '/tmp/x.gdb', '--args'])
```

Comment:
Why aren't we starting this in a new graphical terminal?

Reply:
we do that down below https://github.com/ArduPilot/ardupilot/pull/14363/files/af2fbe4047435c3bdb3bcc987fd3ce92cb18b865#diff-d2de282cca7753bde1b8719902d06dcbR355